src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -1387,14 +1387,8 @@ public ServiceConfig load(ServiceConfig config) {
    * 
    */
   public ServiceConfig getConfig() {
-    ServiceConfig sc = new ServiceConfig();
-    initConfig(sc);
-    return sc;
-  }
-
-  protected ServiceConfig initConfig(ServiceConfig config) {
-    config.type = getSimpleName();
-    return config;
+    // FIXME !!! - this should be null for services that do not have it !
+    return new ServiceConfig();

```

Comment:
this method..  should be abstract..   (or maybe in the short term add a log message saying no config loading exists for this service type here...  as a reminder that we're missing a config class.)

Reply:
long term - with that polished pr i have waiting ... this would only be implemented in Service, and not needed typically in the derived class - so I don't want to make it abstract here...
I don't consider it an error currently for a service not to have config - so I'll log.info it